src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/forgotpassword/AttachedForgotPasswordPage.java

```diff
@@ -0,0 +1,56 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.forgotpassword;
+
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.AuthPageContext;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FormError;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FormPage;
+import lombok.Getter;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'lombok.Getter'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)